tst/components/ui/Modal/ModalButtonGroup.test.tsx

```diff
@@ -111,4 +111,24 @@ describe('ModalButtonGroup', () => {
     expect(component.props.value.onClose).toHaveBeenCalled();
     expect(getByLabelText('close')).toBeInTheDocument();
   });
+
+  it('does not pass the onClose function when "dismissible" is false', () => {
```

Comment:
I think `onClose` is still passable, but I think it should be. I don't think the test is needed.

Reply:
I think I could benefit from renaming this. It should be something like "it doesn't inject 'onClose' into the click handler of the button when "dismissible" is false".

We're not removing the buttons here, just not injecting the onClose.